### ISSUE TITLE
Missing option - don't swipe table cell, just buttons

### ISSUE THREAD

**Issue by mark2b** [MortimerGoro/MGSwipeTableCell#106]
It will be great to add such option.
It is especially useful when you have just right buttons and content of table cell is left aligned. 

Currently I just commented next line of method `[MGSwipeTableCell setSwipeOffset:(CGFloat) newOffset]`
```
//    _swipeView.transform = CGAffineTransformMakeTranslation(newOffset, 0);
```

**Comment by MortimerGoro** [comment]
Thanks for the idea, I have added onlySwipeButtons property to swipe settings

**Comment by MortimerGoro** [comment]
Available in 1.5.1
